crates/dojo-world/src/utils.rs

```diff
@@ -395,6 +395,29 @@ where
     }
 }
 
+impl<T> TransactionExt<T> for AccountDeployment<'_, T>
+where
+    T: AccountFactory + Sync,
+{
+    type R = DeployAccountTransactionResult;
+    type U = AccountFactoryError<T::SignError>;
+
+    async fn send_with_cfg(
+        mut self,
+        txn_config: &TxnConfig,
+    ) -> Result<Self::R, AccountFactoryError<<T>::SignError>> {
+        if let TxnConfig { fee_estimate_multiplier: Some(fee_est_mul), .. } = txn_config {
+            self = self.fee_estimate_multiplier(*fee_est_mul);
+        }
+
+        if let TxnConfig { max_fee_raw: Some(max_raw_f), .. } = txn_config {
+            self = self.max_fee(*max_raw_f);
```

Comment:
Add comment to mention that max raw fee takes precedence on fee estimate multiplied in starknet-rs.

Reply:
i think its okay not to specify that in all of the traits implementation since its already mentioned in the trait definition. wdyt?

https://github.com/dojoengine/dojo/pull/1927/files#diff-96550ff122f6cd6d01f065a5a7e8e0b992a1e3989e2bb6350ccc9b5c950acd6cR345-R349